app/controllers/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+class UsersController < ApplicationController
+
```

Comment:
I would add some sort of authentication on the `destroy` method. Otherwise, if I hit your `destroy` path, I can destroy all your users, since you are not validating that the person is an admin or that the user being destroyed is the current user